docs/compliance/fedramp.md

```diff
@@ -4,8 +4,33 @@ keywords: standards, compliance, security
 title: FedRAMP
 ---
 
-The [Federal Risk and Authorization Management Program (FedRAMP)](https://www.fedramp.gov/) is a U.S. Federal Government-wide program that provides for a standardized approach to security assessment and authorization. Federal agencies that choose to leverage cloud services must ensure that they're utilizing FedRAMP-authorized providers. The security controls FedRAMP requires a provider to adhere to are a subset of the controls documented by NIST Special Publication 800-53. As with the baselines set within NIST 800-53 (low, moderate and high), FedRAMP also incorporates these same baselines in its authorization process. In addition, when agencies deploy systems (like Docker Enterprise Edition) on top of these providers, they must acquire an Authority to Operate (ATO) for those system that are in line with those agencies' own security procedures.
+The [Federal Risk and Authorization Management Program
+(FedRAMP)](https://www.fedramp.gov/) is a U.S. Federal Government-wide program
+that enables a standardized approach to security assessment and authorization.
+Federal agencies that choose to leverage cloud services must ensure that they're
+utilizing FedRAMP-authorized providers. The security controls FedRAMP requires a
+provider to adhere to are a subset of the controls documented by NIST Special
+Publication 800-53. As with the baselines set within NIST 800-53 (low, moderate
```

Comment:
FedRAMP incorporates the NIST 800-53 benchmarks (low, moderate, and high) in its authorization process.

When agencies deploy systems, such as Docker EE, on top of <something better than "these"> providers, they must acquire an ATO for the implementation.

(trying to remove some parenthetical phrases and streamline some text, but I don't want to lose any meaning, so use your judgment)